packages/dd-trace/src/opentelemetry/sampler.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+class Sampler {
+  shouldSample (context, traceId, spanName, spanKind, attributes, links) {
+    // 0 = no, 1 = record, 2 = record and sample
+    // TODO: Make this actually do sampling...
```

Comment:
Yes, this was part of the future work stuff. Got partway into writing it when the scope was finally decided and opted to just leave those bits around and commented out in `tracer.startSpan` for the time being. As it is presently, it will delegate sampling decisions to our existing tracer rather than allowing it to be configurable through OTel.